core/core.php

```diff
@@ -118,28 +128,12 @@ public function get_update_sql($fingerprint)
 		$sql_array = array(
 			'last_accessed' => $timestamp
 		);
+
 		$sql = 'UPDATE ' . $this->table_prefix . 'openra_keys' . '
 			SET ' . $this->db->sql_build_array('UPDATE', $sql_array) . '
 			WHERE fingerprint = "' . $this->db->sql_escape($fingerprint) . '"';
-		return $sql;
```

Comment:
Is this intentional? Seems like a method which has a name that starts with `get_` should return something?